queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
+SELECT name from animals WHERE date_of_birth BETWEEN '2017-05-12' AND '2020-02-02';
```

Comment:
- [ ] Kindly consider updating this query as the dates in your query are wrong. it should be `'2016/01/01' AND '2019/31/12'`  as in the following query :
    -  List the name of all animals born between 2016 and 2019.

Reply:
- [ ] Kindly consider updating this query as the dates in your query are wrong. it should be `'2016/01/01' AND '2019/31/12'`  as in the following query :
    -  List the name of all animals born between 2016 and 2019.
    - Edited the query to match the date range as specified.
    
![Screenshot from 2023-07-10 14-28-44](https://github.com/lincoln1883/animal-hospital/assets/84746035/7e0c479a-c7ae-4223-a4a7-b8e3305e967a)